src/shared/ui/icons/index.tsx

```diff
@@ -119,3 +120,19 @@ export const ChevronDownIcon: FC<IconProps> = props => (
 export const EditIcon: FC<IconProps> = props => (
   <FontAwesome name="edit" {...props} />
 );
+
+export const EyeIcon: FC<IconProps> = props => (
+  <FontAwesome name="eye" {...props} />
+);
+
+export const EyeSlashIcon: FC<IconProps> = props => (
+  <FontAwesome name="eye-slash" {...props} />
+);
+
+export const FeatherCrossIcon: FC<IconProps> = props => (
+  <Feather name="x" {...props} />
+);
+
+export const FeatherCheckIcon: FC<IconProps> = props => (
+  <Feather name="check" {...props} />
+);
```

Comment:
The "Feather" font seems to be causing errors on iOS; I guess you have to add the font there, too?